website-components-playground/package.json

```diff
@@ -12,16 +12,16 @@
   "dependencies": {
     "@commercetools-frontend/application-components": "18.7.0",
     "@commercetools-frontend/i18n": "18.7.0",
-    "@commercetools-uikit/design-system": "^11.0.2",
-    "@commercetools-uikit/icon-button": "^11.2.0",
-    "@commercetools-uikit/icons": "^11.2.0",
-    "@commercetools-uikit/multiline-text-field": "^11.2.0",
-    "@commercetools-uikit/secondary-button": "^11.2.0",
-    "@commercetools-uikit/select-field": "^11.2.0",
-    "@commercetools-uikit/spacings": "^11.2.0",
-    "@commercetools-uikit/text": "^11.2.0",
-    "@commercetools-uikit/text-field": "^11.2.0",
-    "@commercetools-uikit/text-input": "^11.2.0",
+    "@commercetools-uikit/design-system": "^11.2.1",
+    "@commercetools-uikit/icon-button": "^11.2.1",
+    "@commercetools-uikit/icons": "^11.2.1",
+    "@commercetools-uikit/multiline-text-field": "^11.3.0",
+    "@commercetools-uikit/secondary-button": "^11.2.1",
+    "@commercetools-uikit/select-field": "^11.2.1",
+    "@commercetools-uikit/spacings": "^11.2.1",
+    "@commercetools-uikit/text": "^11.2.1",
+    "@commercetools-uikit/text-field": "^11.2.1",
+    "@commercetools-uikit/text-input": "^11.2.1",
```

Comment:
okay we will have conflicts.
I'm fine with merging this PR first then updating ui-kit on top of that.